### ISSUE TITLE
pom.json

### ISSUE THREAD

**Issue by reiz** [stempler/gradle-versioneye-plugin#9]
The [VersionEye Maven Plugin](https://github.com/versioneye/versioneye_maven_plugin) does not simply upload a pom.xml file to the API. It resolves all dependencies locally and creates a `pom.json` file which contains the dependencies with fixed version strings. This `pom.json` gets uploaded to the VersioinEye API to create/update a project. The format of the `pom.json` file is described [here](https://github.com/versioneye/pom_json_format). Would be great if the Gradle plugin would work the same way :-) 

Soon there will be a [VersionEye SBT plugin](https://github.com/versioneye/versioneye_sbt_plugin) which works the same way ;-)

**Comment by stempler** [comment]
This plugin works in the same way (after all there is no pom.xml to start with). The reason I'm creating a  `pom.xml` file and not a `pom.json` is that when I created the plugin I was not able to find out the exact file format needed (I tried some attempts at that, which were all seemingly not correct), which is why I finally resorted to generate a `pom.xml` file from the resolved dependencies because the format was clear there and the VersionEye API finally accepted that.

Would probably not involve much changes to [PomTask](https://github.com/stempler/gradle-versioneye-plugin/blob/master/src/main/groovy/org/standardout/gradle/plugin/versioneye/PomTask.groovy) to generate a Json file instead of an XML file, though I am currently not really using the plugin I must admit and hardly have the time to test it.
